default.nix

```diff
@@ -25,10 +25,10 @@ let
     version = toString timestamp;
     bin = binary-mainnet;
     arch = "amd64";
-    license = "MPL-2.0";
+    license = "OSL";
```

Comment:
AFAIU that license in exactly MIT, so I think it's better to write "MIT" here, it is more recognizable.